crates/sui-protocol-config/src/lib.rs

```diff
@@ -3150,6 +3156,12 @@ impl ProtocolConfig {
 
                     cfg.feature_flags.variant_nodes = true;
                 }
+                73 => {
+                    if chain != Chain::Mainnet && chain != Chain::Testnet {
+                        cfg.consensus_gc_depth = Some(60);
```

Comment:
nit: might be useful for some quick comment about why "60" was chosen as the gc depth